### ISSUE TITLE
[Color system] [a11y] Usability test guidance and implementation for Accessibility

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris-react#2025]
Check in with @dpersing to make sure that this user story is being met.

> As a user of Polaris I can understand how the color should be used based on the name provided so that I can meet WCAG 2.1 and increase future potential by being ready for changes.

This will be a later stage task once we have medium to high confidence on the color system application. It will involve usability testing and making sure that users understand how to use colors to make accessible palettes.

This is also related to the guidance we provide on the color system: https://github.com/Shopify/polaris-react/issues/2031

**Comment by tmlayton** [comment]
@alex-page This issue is a bit vague. Can it be updated to be more specific, updated to be a user story, or split into smaller parts?

**Comment by alex-page** [comment]
Yeah for sure, thanks Tim.

**Comment by tmlayton** [comment]
With the switch to HSLuv, the bulk of the work here is updating Pally to check against WCAG 2.1 or switch to another tool which checks against WCAG 2.1.

We are already building with the new design language changes against WCAG 2.0 which is currently passing.
